FreeOrion does not compile in gcc 5.4.0

# Environment

* **FreeOrion Version**: Git 4195b0c
* **Operating System**: Ubuntu
* gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609

### Description

```
freeorion/UI/CUIControls.cpp: In constructor ‘StatisticIcon::StatisticIcon(std::shared_ptr<GG::Texture>, double, int, bool, GG::X, GG::Y)’:
freeorion/UI/CUIControls.cpp:1304:42: error: converting to ‘const value_type {aka const std::tuple<double, int, bool>}’ from initializer list would use explicit constructor ‘constexpr std::tuple< <template-parameter-1-1> >::tuple(_UElements&& ...) [with _UElements = {double&, int&, bool&}; <template-parameter-2-2> = void; _Elements = {double, int, bool}]’
     m_values(1, {value, digits, showsign})
```

### Expected Result

I expect the source code to compile.

### Steps to reproduce

* Compile the mentioned FreeOrion version with the mentioned compiler.

--- a/UI/CUIControls.cpp
+++ b/UI/CUIControls.cpp
@@ -1301,7 +1301,7 @@ StatisticIcon::StatisticIcon(const std::shared_ptr<GG::Texture> texture,
                              double value, int digits, bool showsign,
                              GG::X w /*= GG::X1*/, GG::Y h /*= GG::Y1*/) :
     GG::Control(GG::X0, GG::Y0, w, h, GG::INTERACTIVE),
-    m_values(1, {value, digits, showsign})
+    m_values(1, std::tuple<double, int, bool>{value, digits, showsign})
 {
     m_icon = GG::Wnd::Create<GG::StaticGraphic>(texture, GG::GRAPHIC_FITGRAPHIC);
 }

